scripts/creation-portal-migration/api-list/headers.js

```diff
@@ -0,0 +1,133 @@
+const { default: axios } = require("axios");
+const { CONFIG } = require("../constant/config");
+const querystring = require("query-string");
+const jwt = require("jsonwebtoken");
+const logger = require("../logger");
+
+/**
+* To generate the user token
+* @method
+* @name genToken
+* @param {String} url - url 
+* @param {querystring} body - body 
+* @param {String} type - type
+* 
+  * @returns {string} - Generates the user token
+*/
+const genToken = async (url, body, type) => {
+  const isValid = await validateToken(type);
+
+  const headers = {
+    "Content-Type": "application/x-www-form-urlencoded",
+  };
+
+  if (!isValid) {
+    const res = await axios.post(url, body, headers).catch((err) => {
+      logger.error(`Error while generateToken Error: ${JSON.stringify(err?.response?.data)}`)
+      return err;
+    });
+    return res ? res?.data?.access_token : "";
+  } else {
+    const token = this.ed_token;
+    // type === "ed" ? this.ed_token : this.ed_token
+    // this.creation_portal_token;
+
+    return token;
+  }
+};
+
+/**
+* To validate the user token
+* @method
+* @name validateToken
+* @param {String} type - type
+* 
+  * @returns {Boolean} - Returns the boolean response  
+*/
+
+const validateToken = (type) => {
+  const token = type === "ed" ? this.ed_token : this.ed_token;
```

Comment:
@jayesh9747 create a new common file inside the scripts/creation-portal-migration/constant.js then add the string. So we want to change the value in future. Only one place we can change. You can refer to the common file in this service.

Reply:
this means.  I Create a constant file that contains only key-value pairs and export it directly, for example: { ED: "ed" } .
right ?

Reply:
Exactly